tests/conftest.py

```diff
@@ -10,6 +10,10 @@
 from beebot.body import Body
 
 
+def pytest_configure():
```

Comment:
This change ensures that the environment variables are loaded before the baserun pytest plugin starts up